osu.Game/Screens/Multi/Play/PlaylistsPlayer.cs

```diff
@@ -20,7 +20,7 @@
 
 namespace osu.Game.Screens.Multi.Play
```

Comment:
Should probably make its way over to
```suggestion
namespace osu.Game.Screens.Multi.Playlists
```